File: FreelancerApp/API/DTOs/ProjectActiveDTO.cs
namespace API.DTOs;

public class ProjectActiveDTO
{
    public int Id { get; set; }
    public string? Title { get; set; }
    public string Description { get; set; } = string.Empty;
    public bool IsAssigned { get; set; }
    public List<string> SkillNames { get; set; } = new();
    public int ClientUserId { get; set; }
    public string? ClientKnownAs { get; set; }
    public string? ClientPhotoUrl { get; set; }
    public int? FreelancerUserId { get; set; }
    public string? FreelancerKnownAs { get; set; }
    public string? FreelancerPhotoUrl { get; set; }
    public string? PhotoUrl { get; set; }
}

File: FreelancerApp/client/src/app/_DTOs/projectActiveDTO.ts
export interface ProjectActiveDTO {
  id: number;
  title?: string;
  description: string;
  isAssigned: boolean;
  skillNames: string[];
  clientUserId: number;
  clientKnownAs?: string;
  clientPhotoUrl?: string;
  freelancerUserId?: number;
  freelancerKnownAs?: string;
  freelancerPhotoUrl?: string;
  photoUrl?: string;
}